helm-values/vc-authn-oidc/dev.yaml

```diff
@@ -0,0 +1,94 @@
+ingressSuffix: -dev.apps.silver.devops.gov.bc.ca
+
+image:
+  repository: ghcr.io/bcgov/vc-authn-oidc
+  pullPolicy: IfNotPresent
+  tag: "2.0.0-beta1"
+
+acapyTenancyMode: single
+setNonRevoked: true
+useOobPresentProof: false
+useOobLocalDIDService: false
+controllerCameraRedirectUrl: wallet_howto
+controllerPresentationExpireTime: 300
+useHTTPS: true
+logLevel: INFO
+
+resources:
+  limits:
+    cpu: 100m
+    memory: 512Mi
+  requests:
+    cpu: 10m
+    memory: 128Mi
+
+autoscaling:
+  enabled: true
+  minReplicas: 1
+  maxReplicas: 2
+  targetCPUUtilizationPercentage: 80
+
+networkPolicy:
+  enabled: true
+  ingress:
+    enabled: true
+    namespaceSelector:
+      network.openshift.io/policy-group: ingress
+
+ingress:
+  enabled: true
+  annotations:
+   route.openshift.io/termination: edge
+
+acapy:
+  image:
+    repository: ghcr.io/hyperledger/aries-cloudagent-python
+    pullPolicy: IfNotPresent
+    tag: py3.9-0.9.0
```

Comment:
Can you please upgrade to the latest `0.1.2` release? Or go with whatever is specified in the main chart.